tests/unit/niclips/figures/test_multi_view.py

```diff
@@ -5,6 +5,7 @@
 import nibabel as nib
 import numpy as np
 import pytest
+from _pytest.logging import LogCaptureFixture
```

Comment:
Is `_pytest` a typo?

Reply:
I'll double check, but I think i grabbed this from another test file where the `LogCaptureFixture` was used.